package.json

```diff
@@ -61,6 +61,8 @@
     "e2e-test-runner-build": "ncc build tests/e2e/testRunner.ts -o tests/e2e/dist/"
   },
   "dependencies": {
+    "@babel/plugin-proposal-private-methods": "^7.18.6",
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```

Comment:
These were required for our jest test and existed as a peer dependency of storybook v6. Since the latest version does not contain this, I added them explicitly as a dependency.